frontend/src/app/leaderboard/table.tsx

```
@@ -0,0 +1,147 @@
+                    <div className="font-bold">You</div>
+                  </div>
+                </div>
+              </td>
+              <td>
+                <span className="">
+                  Community Outreach Specialist
+                </span>
+              </td>
+              <td>Indigo</td>
+              <th>
+                <button className="btn btn-ghost btn-xs">details</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)